content/influxdb/v2.1/security/enable-hardening.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enable security features
+seotitle: Enable security/hardening features
+description: >
+  Enable additional security features in InfluxDB.
+weight: 102
+menu:
+  influxdb_2_1:
+    parent: Security & authorization
+influxdb/v2.1/tags: [security, hardening]
+products: [oss]
+---
+
+InfluxDB is used in many environments and depending on your environment, enabling additional security features may be appropriate.
+To enable all security hardening options now and in the future,
+start InfluxDB with the `--hardening-enabled` command line flag:
+
+    ```bash
```

Comment:
Just a nit -- is it necessary (or conventional) to multi-line these commands when they're so short?